src/main/java/com/epam/reportportal/junit5/ReportPortalExtension.java

```diff
@@ -357,7 +357,8 @@ public void testDisabled(ExtensionContext context, Optional<String> reason) {
 				String rawDescription = createStepDescription(context, itemType);
 				return StringUtils.isNotBlank(rawDescription) ? MarkdownUtils.asTwoParts(r, rawDescription) : r;
 			}).orElse(null);
-			startTestItem(context, Collections.emptyList(), itemType, description, null);
+			String methodName = reason.orElse(createStepMethodName(context, itemType));
```

Comment:
I can't recall, that I asked to change this? What's this for?

Reply:
Removed it.